climada/engine/impact.py

```diff
@@ -813,7 +847,8 @@ def plot_basemap_impact_exposure(self, event_id=1, mask=None, ignore_zero=False,
                                                  buffer, extend, zoom, url, axis=axis, **kwargs)
 
         return axis
-
+    
+    # TODO: replace with subplots_from_gdf()
```

Comment:
The name `subplots_from_gdf` is a bit confusing to me. Why would this be a `subplot` instead of a `plot`?

Reply:
The `subplots_from_gdf` already exists, it is how we now plot local return periods for given hazard intensity thresholds. I named it subplots because it produces a plot with several subplots, each for a different column of the gdf. I am happy to change the name though, just "plot_from_gdf" or "plots_from_gdf"?

Reply:
I would then prefer `plot_from_gdf`. Plot is quite clear, and can contain more than one. Plurals are in my opinion to be avoided as much as possible as they easily create confusion.